### DISCUSSION TITLE
PnP Modern Search Results: RefinablesProperties are empty

### DISCUSSION THREAD

**Discussion by afnepipe** [microsoft-search/pnp-modern-search#2975] [Q&A]
Hi Community,
I appreciate if someone can give me a recommendation on what is happening to me.

As you can see in the images only some of the RefinabableString properties are showing values. 

On May 5th the configuration of the:
- Mapping of properties vs. refinableString (All columns are Text type).
- Reindex Document Library was made in the advanced options page of the library
- And also the ReIndex Site in the Searcg and Offline Availability page.

![image](https://github.com/microsoft-search/pnp-modern-search/assets/96144314/ff1ae2c5-2fbe-4a59-ba87-a35098c365c2)
![image](https://github.com/microsoft-search/pnp-modern-search/assets/96144314/f55219b1-3d41-4e8c-adff-ec53ff6bf91b)

**Comment by wobba** [comment]
Did you add them to the list of properties to be retrieved?
![image](https://github.com/microsoft-search/pnp-modern-search/assets/51104/71d5b3ab-6f27-4a47-ad6a-14c5b5b0fc5a)

**Reply by afnepipe** [reply]
Hi @wobba Thanks for your answer.
Yes. Those are the selected properties.

![image](https://github.com/microsoft-search/pnp-modern-search/assets/96144314/e06554f3-ca7a-4b02-8248-70af36757219)

**Reply by wobba** [reply]
Then you switch to the debug template and verify that your items have a value returned. If not, an indexing issue of some kind.

**Reply by afnepipe** [reply]
Hi @wobba.
These are the results in the debug mode.
![image](https://github.com/microsoft-search/pnp-modern-search/assets/96144314/149c324b-bd60-46f0-bd2b-bf2dd1de57c2)

Yeah! I think is a issue in the indexing. I created a Microsoft support case for it.

**Reply by afnepipe** [reply]
@wobba I have a question. Is it normal that the refinablestring are empty (as if there is no mapping) at the tenant level in the Search page when accessing through -admin.sharepoint.com?

I clarify the mapping configuration was done at the site level.
![image](https://github.com/microsoft-search/pnp-modern-search/assets/96144314/a56c68d3-c716-4cc7-b438-52a5182f8535)

**Reply by kasperbolarsen** [reply]
Absolutely!  Any configuration at Site Collection level will only be visible at that level. Any configuration at the tenant level should be visible at the site collection level, but can be overridden.

**Reply by wobba** [reply]
...which is why I wrote this one https://www.techmikael.com/2016/02/search-schema-mapping-level-explained.html

and I don't like site level mapping if you can avoid it. Too hard to govern and you can have overwrites you don't understand. Like someone mapping a cp with no value - which messes up your tenant level mapping.
